dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/LambdaUrlChecker.java

```diff
@@ -71,6 +77,16 @@ private Optional<Boolean> checkUrl(String url) {
         }
     }
 
+    private Optional<Boolean> checkStatus(String body) {
+        if ("{\"message\":true}".equals(body)) {
+            return Optional.of(true);
+        }
+        if ("{\"message\":false}".equals(body)) {
+            return Optional.of(false);
+        }
```

Comment:
This section tingles my spider sense.  If it's easy, we should probably parse the JSON and access the value of `message` via the parsed representation.  That way, if response formatting changes, it won't mysteriously break.

Reply:
Yeah, fair point, I had the same concerns, but had left it as it was. This was existing code that is only showing up in the PR because I refactored into a new method. I'll update it.